owncloud/owncloud.py

```diff
@@ -731,6 +731,28 @@ def user_exists(self, user_name):
         :raises: ResponseError in case an HTTP error status was returned
 
         """
+        users=search_users(self, user_name)
+
+        if len(users) > 0:
+            for user in users:
+            	if user == user_name:
+            		return True
+
+            return False
+        else:
+        		return False
```

Comment:
Indent seems off here.
I suggest to simply add `return False` at the end of the function.